app/grants/tasks.py

```diff
@@ -206,6 +212,19 @@ def recalc_clr(self, grant_id, retry: bool = True) -> None:
             only_grant_pk=obj.pk
         )
 
+
+@app.shared_task(bind=True, max_retries=1)
+def process_predict_clr(save_to_db=False, from_date=None, clr_round=None, network='mainnet', only_grant_pk=None):
+    from grants.clr import predict_clr
+    return predict_clr(
+        save_to_db=False,
+        from_date=None,
+        clr_round=None,
+        network='mainnet',
+        only_grant_pk=None
+    )
```

Comment:
^ I don't get why we are doing this?

Reply:
Did you test this out @chibie ? 
Could we pass only the needed arguments as is

Reply:
Also do we need a return ?

Reply:
`predict_clr` function in clr.py returns `debug_output`

Reply:
Ah i don't know what we use that for ! 
Cause if you look at other places -> we don't do anything with it

Reply:
could we switch it to 
```
process_predict_clr(save_to_db, from_date, clr_round, network):

    predict_clr(
        save_to_db,
        from_date,
        clr_round,
        network
    )
```